schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/payment/workflow/ActivityPaymentWorkflow.java

```diff
@@ -55,8 +58,9 @@ public void onCreated() throws PaymentWorkflowException {
             ActivityService activityService = ActivityServiceFactory.getActivityService();
             Subscription subscribe = activityService.subscribe(userId, activityId);
             subscriptionId = subscribe.getId();
-        } catch (EntityNotFoundException | OperationException e) {
-            log.error("Failed to subscribe User={} to Activity={}", userId, activityId); // TODO: Replace with event recorder
```

Comment:
I don't think we should remove the logging...  It is still important to have this in the logs I think.